draft-ietf-tls-tls13.md

```diff
@@ -2783,20 +2741,21 @@ can behave in one of two ways:
   process the early data. It is not possible for the server
   to accept only a subset of the early data messages.
 
-Prior to accepting the "early_data" extension, the server
-MUST perform the following checks:
+Prior to accepting the "early_data" extension, the server MUST
+validate that the session ticket parameters are consistent with its
+current configuration. It MUST also validate that the extensions
+negotiated in the previous connection are identical to those being
+negotiated in the ServerHello, with the exception of the
+following extensions:
 
-- The configuration_id matches a known server configuration.
+- The use of "signed_certificate_timestamp" {{!RFC6962}} MUST
+  be identical but the server's SCT extension value may differ.
 
-- The client's cryptographic determining parameters match the
-  parameters that the server has negotiated based on the
-  rest of the ClientHello. If (EC)DHE is selected, this includes
-  verifying that (1) the ClientHello contains a key from the
-  same group that is indicated by the server configuration and
-  (2) that the server has negotiated that group and will
-  therefore include a share from that group in its own
-  "key_share" extension.
+- The "padding" extension {{RFC7685}} MUST be ignored for this purpose.
 
+- The values of "key_share", "pre_shared_key", and "early_data", which MUST
+  be as defined in this document.
```

Comment:
Is the above list the result of a complete analysis.  For instance, I don't know SRP, but is there any sense that the srp extension needs to be the same?

For something I do know, it seems like signature_algorithms could be different without affecting security.